server/views/pages/recommendations/editDateOfBirth.njk

```diff
@@ -11,23 +11,31 @@
         <div class="govuk-grid-column-two-thirds-from-desktop">
             {% include '../../partials/error-summary.njk' %}
 
-            {% set html %}
+            {# {% set html %}
                 <p class="govuk-body">This will update PPUD.  It will not update NOMIS.</p>
+
             {% endset %}
             {{ govukNotificationBanner({
                 html: html,
                 classes: "govuk-notification-banner--info govuk-body"
-            }) }}
+            }) }} #}
             <h1 class='govuk-heading-l'>{{ pageHeadings[page.id] }}</h1>
 
             <form novalidate method="post">
                 <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-
                 <fieldset class="govuk-fieldset" role="group"
                           aria-describedby="date-of-birth-hint appointment-time-hint{% if errors.dateOfBirth %} error{% endif %}">
 
-                    <div id="date-of-birth-hint" class="govuk-hint">
-                        For example, 31 3 1980
+                    <div class="govuk-body">
+                        {% if dateOfBirthNomis and dateOfBirthPpud %}
```

Comment:
Not convinced about this if...else logic.  It will display the "From NOMIS" line if dateOfBirthNomis is null.  Perhaps better to say `if dateOfBirthNomis - display dateOfBirthNomis `then separately,  `if dateOfBirthPppud - display dateOfBirthPpud`. That gives very clear and safe path through the code.  Or... if you want to make sure that "From NOMIS" is always displayed no matter what, don't include dateOfBirthNomis in any if check.

Reply:
Corrected as suggested 👍